### DISCUSSION TITLE
[Feature] Make navigation gesture pill transparent

### DISCUSSION THREAD

**Discussion by M07SRT** [gokadzev/Musify#366] [General]
![Screenshot_2024-03-26-06-55-24-597_com.gokadzev.musify-edit.jpg](https://github.com/gokadzev/Musify/assets/89362987/cf8db826-5bdd-4d34-abaa-b88257c06f34)

Can you make the navigation bar transparent?

**Comment by gokadzev** [comment]
> ![Screenshot_2024-03-26-06-55-24-597_com.gokadzev.musify-edit.jpg](https://private-user-images.githubusercontent.com/89362987/316723622-cf8db826-5bdd-4d34-abaa-b88257c06f34.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0NTQ4MjgsIm5iZiI6MTcxMTQ1NDUyOCwicGF0aCI6Ii84OTM2Mjk4Ny8zMTY3MjM2MjItY2Y4ZGI4MjYtNWJkZC00ZDM0LWFiYWEtYjg4MjU3YzA2ZjM0LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI2VDEyMDIwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlmYWY0MTRkMGUwMDA1MzVkMjY5OTU5YWFhNWFlM2Q2MDJhZTFjODk1MGYxZTMyNDliOGFhNmViMTc2Zjg5MjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FCEV61zFsW-DOYDQGASbS8HHGTWXkCOZDfmNx53KkHc)
> 
> Can you make the navigation bar transparent?

What's wrong with that one? It's material design 3 navbar without any modifications

**Reply by M07SRT** [reply]
![IMG_20240326_174315.jpg](https://github.com/gokadzev/Musify/assets/89362987/98d602d2-772e-4f60-be96-1b4bc462e37a)

Make it like this

**Reply by gokadzev** [reply]
Sorry, not a good idea. (Especially on the dark mode)

**Reply by M07SRT** [reply]
![Screenshot_2024-03-26-20-00-35-250_com.google.android.gm-edit.jpg](https://github.com/gokadzev/Musify/assets/89362987/9d33472c-a681-4c42-9d2d-481cca562b97) I don't think so

**Reply by gokadzev** [reply]
Oh, you mean navigation gesture pill, not navigation bar. Okay, I'll see what I can do about it.

**Comment by gokadzev** [comment]
Will be changed in the next release ✅
